src/Console/ConsoleOutput.php

```diff
@@ -317,9 +319,14 @@ public function getOutputAs()
      *
      * @param int $type The output type to use. Should be one of the class constants.
      * @return void
+     * @throws InvalidArgumentException in case of a not supported output type.
      */
     public function setOutputAs($type)
     {
+        if (in_array($type, [self::RAW, self::PLAIN, self::COLOR], true) === false) {
```

Comment:
I always find !in_array() more readable, no need for the strictness of internal php function checks, right?